jdi-light-material-ui-tests/src/main/java/io/github/com/pages/displaydata/TableFrame.java

```diff
@@ -0,0 +1,24 @@
+package io.github.com.pages.displaydata;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.material.annotations.JDITable;
+import com.epam.jdi.light.material.elements.displaydata.Table;
+import com.epam.jdi.light.material.elements.inputs.Button;
+
+public class TableFrame extends Section {
+    @JDITable
+    public Table table;
+
+    @UI(".MuiIconButton-label[2]")
```

Comment:
suggest this locator "//button[@title='Next page']"
it's more stable because click will be performed directly to button